lib/modules/manager/nuget/update.ts

```diff
@@ -30,8 +31,23 @@ export function bumpPackageVersion(
 
   try {
     const project = new XmlDocument(content);
-    const versionNode = project.descendantWithPath('PropertyGroup.Version')!;
+    const versionNode = findVersion(project);
+    if (!versionNode) {
+      logger.warn(
+        "Couldn't find Version or VersionPrefix in any PropertyGroup"
+      );
+      return { bumpedContent };
+    }
+
     const currentProjVersion = versionNode.val;
+    if (currentProjVersion !== currentValue) {
```

Comment:
note: we need the same change on maven, there it can also fail on same condition

Reply:
Do you want me to make that change as part of this PR?

Reply:
no, should be a separate PR